docs/samples/Microsoft.ML.Samples/Program.cs

```diff
@@ -1,12 +1,15 @@
-﻿using Samples.Dynamic;
+﻿using System.Reflection;
 
 namespace Microsoft.ML.Samples
 {
-    internal static class Program
+    public static class Program
     {
-        static void Main(string[] args)
+        public static void Main(string[] args) => RunAll();
+
+        internal static void RunAll()
         {
-            CalculateFeatureContribution.Example();
+            foreach (var type in Assembly.GetExecutingAssembly().GetTypes())
+                type.GetMethod("Example", BindingFlags.Public | BindingFlags.Static | BindingFlags.FlattenHierarchy)?.Invoke(null, null);
         }
```

Comment:
>} [](start = 8, length = 1)

please add a 1-liner report like "20 sample ran successfully" so that sample-writers get a concrete feedback the the code actually ran